### PR TITLE
[TAS-2162] ⬆️ Add nuxt script and crisp

### PR DESCRIPTION
**Pull request by williamchong**
Requires https://github.com/likecoin/nft-book-press/pull/237

### CODE REVIEW

File: nuxt.config.ts [hunk 1]
```diff
@@ -64,6 +71,11 @@ export default defineNuxtConfig({
 
   runtimeConfig: {
     public: {
+      scripts: {
+        crisp: {
+          id: ''
```
Comment (nwingt) [inline nuxt.config.ts:76]: empty string?

Comment (nwingt) [reply nuxt.config.ts:79]: use `mock`?
https://scripts.nuxt.com/docs/guides/registry-scripts#disabling-in-development

Comment (williamchong) [reply nuxt.config.ts:79]: it probably is not about disabling in dev, but env only works if id is `''`

Comment (williamchong) [reply nuxt.config.ts:79]: e.g. in doc


```
runtimeConfig: {
    public: {
      scripts: {
        cloudflareWebAnalytics: {
          // provide empty string so .env works
          token: '', // NUXT_PUBLIC_SCRIPTS_CLOUDFLARE_WEB_ANALYTICS_TOKEN
        },
      },
    },
  },
```

**Pull request merged**